pkgs/by-name/ii/iio-oscilloscope/package.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
```

Comment:
Please format with the new formatter.

Reply:
Thank you for the hint, @FliegendeWurst . Adjusted accordingly, also rebased to current master.